ample/ensembler/truncation_util.py

```diff
@@ -16,6 +15,14 @@
 
 logger = logging.getLogger(__name__)
 
+MIN_CHUNK = 3 # Theseus needs at least 3 residues in order to work
+
+class TRUNCATION_METHODS(Enum):
+    FOCUSED = 'focussed'
```

Comment:
Although fine these constants don't tend to be uppercase ... but just a small detail